drivers/volume/kdmp/kdmp.go

```diff
@@ -523,8 +522,8 @@ func (k *kdmp) StartRestore(
 		dataExport.Labels = labels
```

Comment:
Not related to this line but for this, can we trim of `-` is its the last char got after truncating? This observation from the issue we saw while testing when names were lengthier and restore was failing.
```
labels[restoreCRNameKey] = truncate.Truncate(restore.Name, labelNamelimit, "", truncate.PositionEnd)
```

Reply:
got it, added it as separate commit